en/components/combo_features.md

```diff
@@ -87,27 +87,134 @@ export class ComboDemo implements OnInit {
 }
 ```
 
-> Note: If [displayKey]({environment:angularApiUrl}/classes/igxcombocomponent.html#displaykey) is omitted then [valueKey]({environment:angularApiUrl}/classes/igxcombocomponent.html#valuekey) entity will be used instead.
+> [!Note]
+> If [displayKey]({environment:angularApiUrl}/classes/igxcombocomponent.html#displaykey) is omitted then [valueKey]({environment:angularApiUrl}/classes/igxcombocomponent.html#valuekey) entity will be used instead.
 
 
 Follow this [topic](combo_remote.md) for more details about binding [igx-combo]({environment:angularApiUrl}/classes/igxcombocomponent.html) with remote data.
 
+
+## Selection
+The combo handles selection differently, depending on its [valueKey]({environment:angularApiUrl}/classes/igxcombocomponent.html#valuekey) property.
+
+### Selection with valueKey
+If a `[valueKey]` is provided, all of the combo's selection events will be handled with the value of the items' `valueKey` property.
```

Comment:
If a `valueKey` is provided

Reply:
as you changed [`valueKey`] to `valueKey` in the sample markup you should change it everywhere in the text

Reply:
I wanted to accent that it was an input by bracing it w/ `[`, but apparently it backfired 🐸 . Thanks for pointing it out, I'll remove the brackets in the text blocks.